### PR TITLE
 #1086 JDK8 Options

### PR DESCRIPTION
**Pull request by zabil**
* Limiting Metaspace size for JDK8
* Adding jruby invoke dynamic to false for rails slowness on jdk8

Relates to #1086.

### CODE REVIEW

File: installers/server/release/server.sh [hunk 1]
```diff
@@ -113,9 +114,11 @@ if [ ! -z $SERVER_LISTEN_HOST ]; then
     GO_SERVER_SYSTEM_PROPERTIES="$GO_SERVER_SYSTEM_PROPERTIES -Dcruise.listen.host=$SERVER_LISTEN_HOST"
 fi
 SERVER_STARTUP_ARGS+=("-server $YOURKIT")
+SERVER_STARTUP_ARGS+=("-XX:+IgnoreUnrecognizedVMOptions")
 SERVER_STARTUP_ARGS+=("-Xms$SERVER_MEM -Xmx$SERVER_MAX_MEM -XX:PermSize=$SERVER_MIN_PERM_GEN -XX:MaxPermSize=$SERVER_MAX_PERM_GEN")
+SERVER_STARTUP_ARGS+=("-XX:MaxMetaspaceSize=$SERVER_MAX_METASPACE")
 SERVER_STARTUP_ARGS+=("$JVM_DEBUG $GC_LOG $GO_SERVER_SYSTEM_PROPERTIES")
-SERVER_STARTUP_ARGS+=("-Duser.language=en -Djruby.rack.request.size.threshold.bytes=30000000")
+SERVER_STARTUP_ARGS+=("-Duser.language=en -Djruby.rack.request.size.threshold.bytes=30000000 -Djruby.compile.invokedynamics=false")
```
Comment (ketan) [inline installers/server/release/server.sh]: This attribute has a typo should be `-Djruby.compile.invokedynamic=false`

File: installers/server/release/server.cmd [hunk 2]
```diff
@@ -66,4 +67,4 @@ set OPTION=-client
...
+"%JAVA_CMD%" %OPTION% %YOURKIT% -XX:+IgnoreUnrecognizedVMOptions -Xms%SERVER_MEM% -Xmx%SERVER_MAX_MEM% -XX:PermSize=%SERVER_MIN_PERM_GEN% -XX:MaxPermSize=%SERVER_MAX_PERM_GEN% -XX:MaxPermSize=%SERVER_MAX_PERM_GEN% -XX:MaxMetaspaceSize=%SERVER_MAX_METASPACE% %JVM_DEBUG% %GC_LOG% %GO_SERVER_SYSTEM_PROPERTIES% -Duser.language=en -DJAVA_SYS_MON_TEMP_DIR="%SERVER_DIR%\tmp" -Dorg.eclipse.jetty.server.Request.maxFormContentSize=30000000 -Djruby.rack.request.size.threshold.bytes=30000000 -Djruby.compile.invokedynamics=false -Duser.country=US -Dcruise.config.dir="%SERVER_DIR%\config" -Dcruise.config.file="%SERVER_DIR%\config\cruise-config.xml" -Dcruise.server.port=%GO_SERVER_PORT% -Dcruise.server.ssl.port=%GO_SERVER_SSL_PORT% -jar "%SERVER_DIR%\go.jar"
```
Comment (ketan) [inline installers/server/release/server.cmd]: is `-XX:IgnoreUnrecognizedVMOptions` a JVM option that is supported by all popular JVMs running on windows/osx linux and *ahem* solaris?

File: installers/server/release/server.sh (hunk 1 above)
Comment (zabil) [inline installers/server/release/server.sh]: Thought so but went by this comment.
https://github.com/jruby/jruby/issues/1858#issuecomment-50159599

It is ```invokedynamic``

**Comment by zabil** [comment]
Rethinking this one

**Pull request closed without merging**